src/routes/startups/+page.svelte

```diff
@@ -23,6 +24,10 @@
     import CommunitySupportChat from '$lib/components/CommunitySupportChat.svelte';
     import { trackEvent } from '$lib/actions/analytics';
     import { Button } from '$lib/components/ui';
+    import CaseStudiesLight from './case-studies-light.svelte';
+    import Features from '$routes/(marketing)/(components)/features.svelte';
+    import Benefits from '$routes/(marketing)/(components)/benefits.svelte';
+    import StartupPartnerCTA from './(components)/StartupPartnerCTA.svelte';
```

Comment:
use an `index.ts` for these and import like `import { ... } from '$routes/(marketing)/(components)'`.